### ISSUE TITLE
Feature Request - User Accessibility Settings

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#18]
Feature Request - User Accessibility Settings - opening for Kolibri as [original KA Lite issue](https://github.com/learningequality/ka-lite/issues/3921) is not going to happen within the scope of KA Lite.
#### Examples of possible options
- [ ] **High Contrast & Background** (see the [original issue](https://github.com/learningequality/ka-lite/issues/3921) for screenshots) @jtamiace @66eli77
  It would be awesome to have, but if the contrast is setup properly from the get go, can be an enhancement feature. 
- [ ] **Show only videos with available subtitles** ([old issue](https://github.com/learningequality/ka-lite/issues/3889), turns out it's a feature regression)
  Very useful for DHOH (deaf and hard on hearing) who **must have** subtitles.
- [ ] **Show videos in English with subtitles in user's target language**
  See the [original KA Lite issue](https://github.com/learningequality/ka-lite/issues/3913) for details. In KA Lite this is valid for DHOH users who would use non-English UI, but would be better served if they view the original video in English, with the translated subs in their own language. Implementation for other channels would depend on how the video content is transcribed & translated.
- [ ] More to come...

**Comment by MCGallaspy** [comment]
@66eli77 @aronasorman 
> Show only videos with available subtitles (there was an original issue at some point?)
>  Show videos in English with subtitles in user's target language

Is there a programmatic way in the core content app to indicate whether a ContentMetadatum has subtitles available? And more generally, whether it has subtitles available in a given language? That could be the first feature request. Also something to consider when we design the front-end.

**Comment by mascot6699** [comment]
I can research about this! Can you elaborate on the format of these videos? (i.e .MKV or .mp4)

**Comment by MCGallaspy** [comment]
Hi @mascot6699! I'm happy to elaborate. Be sure to [join our mailing list and pop into our hipchat room](https://github.com/learningequality/kolibri#how-can-i-contribute) for discussion and guidance. For subtitle availability on content, I have a preliminary question, because I'm sadly not too familiar with subtitling videos. (Perhaps @rtibbles can provide some insight.)

Can optional subtitles be represented using our [core content data structures](https://docs.google.com/document/d/1s8kqh1NSbHlzPCtaI1AbIsLsgGH3bopYbZdM1RzgxN8/edit#heading=h.m06dkjutqqdl) (i.e, ContentMetadata, Files, and Formats)? For instance, if some specific subtitle file type can be used for a variety of *video* formats, then we could associate an optional File object to a Format.

In other words, can a video (regardless of its format) be subtitled using some otherwise independent subtitle file?

**Comment by indirectlylit** [comment]
migrating to Trello since this requires some planning: https://trello.com/c/uBEKiNG8/722-user-a11y-settings

**Comment by radinamatic** [comment]
Some *more* planning... :smile:

**Comment by rtibbles** [comment]
This has been resolved with recent additional metadata work!

**Comment by radinamatic** [comment]
Fishing in the deep pond I see, but 5 years old issue closed still feels like a milestone... 👏🏽
